<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>921</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QToolButton" name="pButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>60</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索作者</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tButton">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>60</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索题目</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>1921</width>
      <height>1051</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
     <item row="0" column="5">
      <widget class="QWidget" name="widget_3" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>300</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>32</number>
       </attribute>
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Year</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Author</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="background">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Key</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QWidget" name="widget" native="true"/>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>60</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TOP100作者</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>250</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查找合作者</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TTbutton">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TOP10年度热词</string>
    </property>
   </widget>
   <widget class="QToolButton" name="titlefuzzy">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>模糊搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1840</x>
      <y>60</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>聚团分析</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuwenjian">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actiondblp_x_l"/>
   </widget>
   <addaction name="menuwenjian"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiondblp_x_l">
   <property name="text">
    <string>dblp.xml</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
